### PR TITLE
[Web] Theme-related fixes

### PR DESCRIPTION
**Pull request by rudream**
## Purpose

This PR fixes some small issues related to the new theme changes that were discovered during the test plan.

`e` counterpart: https://github.com/gravitational/teleport.e/pull/1204

## Implementation

1. 5065f038006763e4822b9fdede5d342557f3bce7 & 0ffb345e9069fa47c24a568eb1877f4495f23684 - Updates the text variables in our theme to match those in the design document. The design document had some updated text opacities and I had forgotten to update them when adding the new themes. The bulk of the files changed in this PR are just from the renamings and snapshot updates throughout the codebase. 

`text.muted` is now being used for the placeholder texts in forms, it is higher contrast than it used to be (24% -> 54%). This had to be done as the low opacity we used for our old placeholders was too low contrast and would fail contrast recommendations from WCAG.

2. c650017db69904c98baf5809c5ca68ca6d8d4188 - Fixes the 'X' button being invisible in the default state on the event range picker calendar.

3. 6d4e5d4e4226b69911ae2b2369dccebd645a2a0e - The default button text opacity in the dark theme was 87% for some reason, this updates it to make it full opacity.

4. cbc78b62ad24abed44a0b3fa6f32f064f5483867 - Adds more visible hover and focus button states for the Passwordless button in the Web UI and Connect.

5. a399059d9585e370b9d65d03f65715a889a5fb70 - Fixes the title text colours in the AWS Console animated screens in dark theme. The background of those titles is dark, and when using light theme it was using the main text colour (black) and the text was blending in with the background.

### CODE REVIEW

**Review by gzdunek — APPROVED** [review]
I found two small problems but other than that looks good.

**Review by ryanclark — APPROVED** [review]

File: web/packages/design/src/theme/darkTheme.ts
Comment (ravicious) [inline web/packages/design/src/theme/darkTheme.ts:1]: Barging in here to report that the spinner in `MenuLogin` is a tad too close in color to the background, is this something that you've seen already? Check out the center element in the MenuLogin story.

spinner in MenuLogin

Comment (rudream) [reply web/packages/design/src/theme/darkTheme.ts:1]: Fixed. Not sure why we were using the brand colour for that spinner as opposed to just the default color.

image
image

Comment (ravicious) [reply web/packages/design/src/theme/darkTheme.ts:1]: That looks closer to how it was I think! I'd probably experiment with increasing the contrast even more.

Comment (ravicious) [reply web/packages/design/src/theme/darkTheme.ts:1]: Ah, I left a comment just after you added another one. :D That looks much better, the contrast checker says that the contrast could be better, but this element is not text and it spins so it _should_ be fine. In any case, it looks good on my old display.

**Comment by rudream** [comment]
88799338715768157ecbfac13ed9f1e643e1cca0 - Fixes https://github.com/gravitational/teleport/issues/25162

**Comment by rudream** [comment]
2230dc4598f613cec8f569da3220a45a632f8b75 - Fixes https://github.com/gravitational/teleport/issues/25162

**Review by ryanclark — APPROVED** [review]

**Pull request merged**